examples/gno.land/p/demo/uuid/uuid.gno

```diff
@@ -0,0 +1,87 @@
+// Copyright 2023 The godruoyi/gno-snowflake Author. All rights reserved.
+// Use of this source code is governed by a MIT.
+
+package uuid
+
+import (
+	"errors"
+	"time"
+)
+
+// These constants are the bit lengths of snowflake ID parts.
+const (
+	TimestampLength uint8  = 41
+	MachineIDLength uint8  = 10
+	SequenceLength  uint8  = 12
+	MaxSequence     uint16 = 1<<SequenceLength - 1
+	MaxMachineID    uint64 = 1<<MachineIDLength - 1
+
+	machineIDMoveLength = SequenceLength
+	timestampMoveLength = MachineIDLength + SequenceLength
+)
+
+// default start time is 2008-11-10 23:00:00 UTC, why ? In the playground the time begins at 2009-11-10 23:00:00 UTC.
+// It can run on golang playground.
+// default machineID is 0
+var (
+	MachineID     uint64 = 0
+	sequence      uint16 = 0
+	StartTime            = time.Date(2008, 11, 10, 23, 0, 0, 0, time.UTC)
+	LastTimestamp int64  = 0
+	LastSequence  uint32 = 0
```

Comment:
It seeme like `LastSequence` does not used.

Reply:
Indeed, I had forgotten to remove it after my tests for the resolver. Thanks